# Single keyboard key detection
string class_name
int32 class_id
float32 confidence
float32 x1
float32 y1
float32 x2
float32 y2
